src/Compilers/CSharp/Test/Semantic/Semantics/IteratorTests.cs

```diff
@@ -35,12 +36,16 @@ IEnumerable<int> I()
             var comp = CreateCompilation(text);
 
             var i = comp.GetMember<MethodSymbol>("Test.I");
+            var publicI = i.GetPublicSymbol();
```

Comment:
>var [](http://example.com/codeflow?start=12&length=3)

Please use explicit type to make it clear that we are testing an interface. In other similar places too.